### PR TITLE
fop 2.2

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fop.rb [hunk 1]
```diff
@@ -20,8 +14,13 @@ class Fop < Formula
   end
 
   def install
+    rm_rf Dir["fop/*.bat"] # Remove Windows files.
     libexec.install Dir["*"]
-    bin.write_exec_script libexec/"fop"
+
+    executable = libexec/"fop/fop"
+    executable.chmod 0555
```
Comment (dunn) [inline Formula/fop.rb]: Huh, why does it need to be set to this?

Comment (DomT4) [inline Formula/fop.rb]: Presume upstream screwed up the permissions on the shell script in this release, given it worked fine before.

Comment (dunn) [inline Formula/fop.rb]: I would expect `0755`, though.

Comment (DomT4) [inline Formula/fop.rb]: Homebrew fixes to [0555 I believe](https://github.com/Homebrew/brew/blob/c9c7f462d37500549127efba96c7a25e5c70de4a/Library/Homebrew/cleaner.rb#L102-L105)?

Comment (DomT4) [inline Formula/fop.rb]:
```
gstat -c "%a %n" /usr/local/opt/openssl/bin/openssl
555 /usr/local/opt/openssl/bin/openssl

gstat -c "%a %n" /usr/local/opt/botan/bin/botan
555 /usr/local/opt/botan/bin/botan
```
etc. I know `0755` tends to be used in formulae but I got the impression Homebrew corrected to `0555` behind the scenes.

Comment (dunn) [inline Formula/fop.rb]: TIL!

**Review by ilovezfs — CHANGES_REQUESTED** [review]
Let's get the permissions fixed upstream before merging this.

**Comment by DomT4** [comment]
I mean that's your choice but... https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-5661 😓.

**Comment by JCount** [comment]
https://issues.apache.org/jira/browse/FOP-2708

**Comment by DomT4** [comment]
Thanks @JCount, appreciate you taking the time 🙇. Was on my list but wanted to tidy up the couple things I highlighted in Mike's Rubocop PR in core first before I forgot those, heh.

**Review by ilovezfs — APPROVED** [review]

**Comment by JCount** [comment]
no more :ant:

**Comment by DomT4** [comment]
Thanks!

**Pull request closed without merging**